spec/models/github_repository_spec.rb

```diff
@@ -150,6 +150,44 @@
       end
     end
 
+    context "importable" do
+      describe "#importing?", :vcr do
```

Comment:
I see what you mean. What I was going for here is to ensure that the other states are also not true.